addons/fire/functions/fnc_burn.sqf

```diff
@@ -0,0 +1,352 @@
+#include "script_component.hpp"
+#include "\z\ace\addons\medical_engine\script_macros_medical.hpp"
+/*
+ * Author: Brandon (TCVM)
+ * Makes object catch fire. Only call from events. Local effects only
+ * Arbitrary values to ignite people. Assumed maximum is "10"
+ *
+ * Arguments:
+ * 0: Vehicle <OBJECT>
+ * 1: Intensity of fire <NUMBER>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [player, 4] call ace_fire_fnc_burn
+ *
+ * Public: No
+ */
+#define INTENSITY_LOSS 0.03
```

Comment:
Leave an empty line below header (all files).

Reply:
Do you mean between the header and the `params` array? Or just between header and macros?

Reply:
Primarily header and macros or params. Between macros and params, not sure, probably yes.